actor/src/main/scala/org/apache/pekko/dispatch/Mailboxes.scala

```diff
@@ -169,7 +169,7 @@ private[pekko] class Mailboxes(
       dispatcherConfig.hasPath("mailbox-type") &&
       dispatcherConfig.getString("mailbox-type") != Deploy.NoMailboxGiven
 
-    // TODO remove in 2.3
+    // TODO remove in Akka 2.3
```

Comment:
This comment should be rephrased, this project is no longer Akka

Reply:
I think that  fine, will do several rounds  of cleaning in 1.1.x。

Reply:
when we agree what Pekko versions to remove things in, we can change this - before to leave the '2.3' text there is confusing